Templates/E-Commerce/Magento/Magento to Dynamics365 - Customers/long-description.md

```diff
@@ -0,0 +1,25 @@
+This template fetches customers from Adobe Commerce / Magento and inserts or updates them to Dynamics365 as contacts.
+Using this template you can synchronize customers from Adobe Commerce / Magento to Dynamics365.
+
+![Template](assets/Magento-to-Dynamics365-Customers.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Magento user should be eligible to obtain an admin authorization token from Magento.
+- The Dynamics365 user should have accesss to the client and tenant IDs and the client secret.
+
+# Implementation and Usage Notes
+
+This template fetches all customers from Magento, and then either inserts or updates them as Contacts in Dynamics365. To determine if a Magento customer already has a corresponding Contact in Dynamics365, the email addresses of the Magento customers are compared to those of the existing Contacts in Dynamics365.
+
```

Comment:
"Contacts" should be lowercase here, even though it is an object in Dynamics365.

Reply:
Also, note the typo which the bot noticed!